decidim-accountability/spec/events/decidim/accountability/proposal_linked_event_spec.rb

```diff
@@ -71,4 +71,22 @@
       expect(subject.resource_text).to eq translated(resource.description)
     end
   end
+
+  describe "hidden_resource?" do
+    context "when resource is not moderated" do
+      let(:resource) { create(:proposal) }
+
+      it "returns false" do
+        expect(subject.hidden_resource?).to be false
+      end
+    end
+
+    context "when resource is moderated" do
+      let(:resource) { create(:proposal, :moderated) }
+
+      it "returns false" do
```

Comment:
Seems like the description title doesn't correspond with the code

```suggestion
      it "returns true" do
```